scripts/preprocessing/04a-run_ica.py

```diff
@@ -59,7 +68,11 @@ def filter_for_ica(raw, subject, session):
     """Apply a high-pass filter if needed."""
     if config.ica_l_freq == config.l_freq or config.ica_l_freq is None:
         # Nothing to do here!
-        msg = 'Not applying high-pass filter.'
+        msg = 'Not applying high-pass filter '
+        if config.ica_l_freq == config.l_freq:
```

Comment:
```suggestion
        if config.ica_l_freq <= config.l_freq:
```

Reply:
you should high pass filtered higher if needed

Reply:
Good catch, I have addressed this in 63dc25a